sdk/program/src/address_lookup_table_account.rs

```diff
@@ -4,10 +4,8 @@
 //!
 //! [v0]: crate::message::v0
 
-use solana_program::pubkey::Pubkey;
-
-#[derive(Debug, PartialEq, Eq, Clone)]
-pub struct AddressLookupTableAccount {
-    pub key: Pubkey,
-    pub addresses: Vec<Pubkey>,
-}
+#[deprecated(
+    since = "1.17.0",
+    note = "Please use `solana_sdk::address_lookup_table::AddressLookupTableAccount` instead"
+)]
+pub use crate::address_lookup_table::AddressLookupTableAccount;
```

Comment:
kill the module file in favor of `pub mod address_lookup_table_account { }` in lib.rs, then deprecate the whole module?